modules/ROOT/pages/files.adoc

```diff
@@ -0,0 +1,54 @@
+== Files
```

Comment:
`Managing Files` for consistency with the index.

Reply:
Renamed header as well as file itself and updated nav